### PR TITLE
Fix stale wallet info

### PR DESCRIPTION
**Pull request by Velenir**
Get new WalletInfo right away

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr652--dexreact.review.gnosisdev.com

File: src/components/TradeWidget/index.tsx [hunk 1]
```diff
@@ -570,7 +570,21 @@ const TradeWidget: React.FC = () => {
       })
     } else {
       // Not connected. Prompt user to connect his wallet
-      await connectWallet()
+      const walletInfo = await connectWallet()
+      if (walletInfo && walletInfo.userAddress && walletInfo.networkId) {
```
Comment (W3stside) [inline src/components/TradeWidget/index.tsx]: `if (walletInfo?.userAddress && walletInfo?.networkId) { ...`

**Comment by Velenir** [comment]
@alfetopito already did good!

File: src/components/TradeWidget/index.tsx (hunk 1 above)
Comment (alfetopito) [reply src/components/TradeWidget/index.tsx]: Hmmm nah. TS warning:

```
Property 'networkId' does not exist on type 'false | WalletInfo'.
  Property 'networkId' does not exist on type 'false'.ts(2339)
```

**Pull request closed without merging**